src/help/guides/getting-started.md

```diff
@@ -65,7 +65,7 @@ This requires iOS 13.3+ or iPadOS 13.4+, the app will CRASH otherwise. Paperback
 ---
  
 ### Paperback Installation
-1. **On Mac**: Open **Mail**.
+1. **On Mac**: Open **Mail** and **Altserver**.
```

Comment:
AltServer should have a conistent capitalization

Reply:
Exact! More precisely it should be `AltServer` (https://altstore.io/)

Reply:
Sorry, I did not understand what you were saying. You were totally right, I misread your sentence.